libcxx/docs/Status/RangesMajorFeatures.csv

```diff
@@ -1,5 +1,6 @@
 Standard,Name,Assignee,CL,Status
```

Comment:
I would actually like us to consider dropping these CSVs instead. This was useful when we implemented the One Ranges proposal since that was a single huge paper with many steps, but I feel like our normal status page with the `ranges` tag contains sufficient information. What would you think about that?

Reply:
Basically I'd like to get rid of this duplicated information.

Reply:
+1 I like dedicated status pages for large papers to keep track of the parts that are and are not implemented.

Reply:
@ldionne @mordante In that case I also propose to remove the Spaceship status pages after the paper is complete, all related open PRs merged and LLVM19 branched.

I updated this PR and description accordingly.

Reply:
As mentioned https://github.com/llvm/llvm-project/pull/99375#discussion_r1685374631 I like to keep spaceship around a few days longer. I dislike the entire removal of the ranges page. The paper https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2021/p2321r2.html adds `zip` which is done, the same paper also adds `zip_transform` which is not done. 

Sorry if my +1 was unclear I meant not adding more papers unless there is a strong need to do so.
The paper `P1068R10` seems small enough to be handled in one PR instead of a set of PRs.

So like only status pages when they carry their weight and not just duplicate information.
(There might be better ways to keep track of the detailed paper status, but this is what we currently have.)

Maybe we can change this PR to only remove the Spaceship status and merge that next week after branching LLVM-20.

Reply:
The `zip` P2321R2 project has a separate more detailed status page: https://libcxx.llvm.org/Status/Zip.html

Sorry If I misunderstood something.

Reply:
I totally forgot we have the dedicated Zip page. I've no idea why `zip(_transform)` and `adjacent(_transform)` are on the ranges page and not on the Zip page :-/

Maybe it would be a good idea to clean up this status page after we've branched and remove all completed papers including their "sub parts".